files/scripts/syncd.sh

```diff
@@ -101,8 +101,13 @@ start() {
         fi
 
         if [[ x"$WARM_BOOT" != x"true" ]]; then
-            /bin/systemctl stop pmon
-            /usr/bin/hw-management.sh chipdown
+            if [[ x"$(/bin/systemctl is-active pmon)" == x"active" ]]; then
+                /bin/systemctl stop pmon
+                /usr/bin/hw-management.sh chipdown
+                /bin/systemctl restart pmon
```

Comment:
what's the pmon behavior on other platforms?  what's the expectation for pmon during warm reboot?

Reply:
@keboliu no impact on warm boot (no chip FW reset is performed). On other platforms there is no service restart - this is Mellanox specific behaviour due to HW-MGMT implementation.